crates/rolldown_common/src/inner_bundler_options/types/minify_options.rs

```diff
@@ -80,12 +80,12 @@ pub struct MinifyOptionsObject {
   pub compress: bool,
   pub remove_whitespace: bool,
 }
-
-impl From<&MinifyOptionsObject> for oxc::minifier::MinifierOptions {
-  fn from(value: &MinifyOptionsObject) -> Self {
-    Self {
-      mangle: value.mangle.then(MangleOptions::default),
-      compress: Some(oxc::minifier::CompressOptions::default()),
+impl MinifyOptionsObject {
+  pub fn to_oxc_minifier_options(&self, target: ESTarget) -> oxc::minifier::MinifierOptions {
+    oxc::minifier::MinifierOptions {
+      mangle: self.mangle.then_some(MangleOptions { top_level: true, debug: false }),
+      compress: Some(CompressOptions { target, drop_debugger: true, drop_console: true })
```

Comment:
@IWANABETHATGUY Can we set `drop_console: false`? It breaks the tests in Vite as it expects the console calls to be preserved. Also I think it's not expected to be set by default.

Reply:
Of course,

Reply:
I will fixed it after fixing another issue.

Reply:
Thanks! made at #3637